lib/mix/tasks/kitto.server.ex

```diff
@@ -3,7 +3,7 @@ defmodule Mix.Tasks.Kitto.Server do
   require Logger
 
   @watchers webpack: [bin: "./node_modules/.bin/webpack-dev-server",
-                      opts: ["--stdin", "--progress"]]
+                      opts: ["--watch", "--progress", "--colors"]]
```

Comment:
I cannot find the `--watch` and `--colors` options.
There's `--color` though, which appears in
https://github.com/webpack/webpack-dev-server/blob/v2.2.0/bin/webpack-dev-server.js#L91

is not compatible with webpack <= 2.0.0 will you also push changes to bump https://github.com/kittoframework/kitto/blob/master/installer/templates/new/package.json#L25